core/templates/dev/head/footer_js_libs.html

```diff
@@ -1,5 +1,7 @@
 {% if DEV_MODE %}
   <script src="/third_party/generated/dev/js/third_party.js"></script>
+  <script src="https://npmcdn.com/angular-toastr/dist/angular-toastr.tpls.js"></script>
```

Comment:
Should make these part of third_party.js -- see manifest.json.

/cc @MAKOSCAFEE, who is a good person to ask if you have any trouble with this.